plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -59,6 +59,110 @@ const {
   xAxisShowMinLabel,
   xAxisShowMaxLabel,
 } = DEFAULT_FORM_DATA;
+
+const appContainer = document.getElementById('app');
+const bootstrapData = JSON.parse(appContainer?.getAttribute('data-bootstrap') || '{}');
```

Comment:
I think that we usually parse bootstrapped data in the main index file of a given view. Do you think we could move those 2 lines higher in the files hierarchy as well?